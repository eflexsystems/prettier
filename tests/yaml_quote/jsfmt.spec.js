run_spec(__dirname, ["yaml"]);
run_spec(__dirname, ["yaml"], { singleQuote: true });
