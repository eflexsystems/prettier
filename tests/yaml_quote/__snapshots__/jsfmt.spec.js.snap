// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quote.yml 1`] = `
- "123"
- '123'
- "''"
- '""'
- ''''
- "\\"\\""
- '\\n123'
- "\\n123"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- "123"
- "123"
- "''"
- '""'
- "'"
- '""'
- '\\n123'
- "\\n123"

`;

exports[`quote.yml 2`] = `
- "123"
- '123'
- "''"
- '""'
- ''''
- "\\"\\""
- '\\n123'
- "\\n123"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- '123'
- '123'
- "''"
- '""'
- "'"
- '""'
- '\\n123'
- "\\n123"

`;
